network/p2p/gossip/bloom.go

```diff
@@ -0,0 +1,129 @@
+// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package gossip
+
+import (
+	"encoding/binary"
+	"hash"
+	"time"
+
+	bloomfilter "github.com/holiman/bloomfilter/v2"
+	"golang.org/x/exp/rand"
+)
+
+const hashLength = 32
+
+var _ hash.Hash64 = (*hasher)(nil)
+
+type Hash [hashLength]byte
```

Comment:
Can we just use `ids.ID` for this? Not sure we need to make a new type for this

Reply:
I originally did this to try to make the gossip package as self-contained as possible but this was probably overly worrisome. `ids` makes sense to pull in.